src/utils/config.py

```diff
@@ -0,0 +1,59 @@
+"""This module contains utility functions that are used internally by the Python Client and the CLI, reliant on the config module"""
+
+from urllib.parse import urljoin
+
+from beartype import beartype
+from beartype.typing import TypeVar
+from deeporigin.config import get_value
+from deeporigin.utils.constants import ObjectType
+
+T = TypeVar("T")
+
+
+@beartype
+def construct_resource_url(
+    *,
+    name: str,
+    row_type: ObjectType,
+) -> str:
+    """Constructs the URL for a resource
+
+    Args:
+        name (str): name of the resource
+        row_type (ObjectType): type of the resource
+
+    Returns:
+        str: URL for the resource
+    """
+
+    env = get_value()["env"]
+    org = get_value()["organization_id"]
+    if env == "prod":
+        url = f"https://os.deeporigin.io/org/{org}/data/{row_type}/{name}"
+    else:
+        url = f"https://os.{env}.deeporigin.io/org/{org}/data/{row_type}/{name}"
```

Comment:
It'd be nice to unify this code.

Reply:
oooof good catch! this needs to use the method i wrote in the last PR

Reply:
fixed